### ISSUE TITLE
Changes in non-exposed modules should be ignored

### ISSUE THREAD

**Issue by gelisam** [kowainik/policeman#56]
For example, when I run `policeman --previous "0.9.0.2"` on https://github.com/haskell-hint/hint/tree/68af39a21acca00ce9ea692709095c4ea2bda07e, I get:

```
  Per module diff:
    * Hint.GHC
      Added exports:
        ExportedFunction "combineSrcSpans"
        ExportedFunction "vcat"

🔄 Type of change:        Minor
👵 Previous version:      0.9.0.2
💎 Suggested new version: 0.9.1.0
```

However, since `Hint.GHC` is listed [in the `other-modules` section](https://github.com/haskell-hint/hint/blob/68af39a21acca00ce9ea692709095c4ea2bda07e/hint.cabal#L83), not the `exposed-modules` section, I would have expected the type of change to be `NoChange`.

**Comment by chshersh** [comment]
Hi @gelisam, thanks for using `policeman` and providing feedback! We do have a function to extract and work on only `exposed-modules`:

https://github.com/kowainik/policeman/blob/0c3f8e4ba845eb7e10c03b951477b9119c82e79d/src/Policeman/Cabal.hs#L80-L85

Need to look closely why `policeman` tries to look in `other-modules` :thinking:

**Comment by gelisam** [comment]
Turns out that's only used to determine whether the list of exposed modules has changed, but you're still comparing the changes in every module. I'll open a PR soon.

**Comment by vrom911** [comment]
Closed in #58
